src/Microsoft.IdentityModel.Logging/LogHelper.cs

```diff
@@ -291,7 +289,7 @@ public static void LogVerbose(string message, params object[] args)
             if (IdentityModelEventSource.Logger.IsEnabled())
                 IdentityModelEventSource.Logger.WriteVerbose(message, args);
 
-            if (Enum.IsDefined(typeof(EventLogLevel), (int)EventLevel.Verbose) && Logger.IsEnabled((EventLogLevel)EventLevel.Verbose))
+            if (Logger.IsEnabled(EventLogLevel.Verbose))
                 Logger.Log(WriteEntry((EventLogLevel)EventLevel.Verbose, null, message, args));
```

Comment:
Can we directly use EventLogLevel.Verbose here as well?

Reply:
Yup